tests/bsim/bluetooth/mesh/tests_scripts/proxy_sol/sol_beacon_rcvd.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Copyright 2024 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+source $(dirname "${BASH_SOURCE[0]}")/../../_mesh_test.sh
+
+# Test procedure:
+# 1. Initialize and configure tester and iut instances.
+# 3. Tester sends a solicitation PDU.
+# 4. Tester scans for beacons and expects to receive at least one private net ID advertisement.
+overlay=overlay_pst_conf_overlay_gatt_conf_overlay_psa_conf
```

Comment:
There should be a test without PSA (same for SRPL test).

Reply:
both with and without PSA (just look at replay cache for example)